include/snitch/snitch_config.hpp.config

```
@@ -97,6 +100,12 @@
 #    define SNITCH_WITH_EXCEPTIONS 0
 #endif
 
+#if defined(SNITCH_WITH_MULTITHREADING)
```

At this stage, the macro is always defined, but with value 0 or 1, so we don't want to just check for `defined(...)`
```suggestion
#if SNITCH_WITH_MULTITHREADING
```